app/controllers/tradingName/CheckYourAnswersController.scala

```diff
@@ -50,9 +53,17 @@ class CheckYourAnswersController @Inject() (checkYourAnswersView: CheckYourAnswe
   def updateTradingName(): Action[AnyContent] = (authPredicate andThen inFlightTradingNamePredicate) { implicit user =>
 
     user.session.get(prepopulationTradingNameKey) match {
-      case Some(_) =>
-            Redirect(controllers.routes.ChangeSuccessController.tradingName())
-              .addingToSession(tradingNameChangeSuccessful -> "true", inFlightTradingNameChangeKey -> "true")
+      case Some(prepopTradingName) =>
+        auditingService.audit(ChangedTradingNameAuditModel(
+          user.session.get(validationTradingNameKey),
```

Comment:
The Confluence page states that "If the user does not have a current trading name then that field is omitted.". Right now this event would audit a blank string for the add journey (i.e. the value in `validationTradingNameKey` would be `Some("")`). Can you add a little logic here to cater for this